build-logic/src/main/groovy/org.apache.groovy-asciidoctor.gradle

```diff
@@ -53,14 +53,14 @@ asciidoctor {
     sources {
         include '*.adoc'
     }
-    outputDir = "$buildDir/asciidoc/html5"
+    outputDir = layout.buildDirectory.dir("asciidoc/html5")
     resources {
         duplicatesStrategy = DuplicatesStrategy.WARN
 
         from("${rootProject.projectDir}/src/spec/doc/assets") {
             include 'css/style.css'
         }
-        from("$buildDir/tmp/asciidoctor-assets")
+        from layout.buildDirectory.dir("asciidoc/html5/assets")
```

Comment:
Should this be the same as the next change, i.e. with tmp in the path?

Reply:
Ooops, typo here